pos_picking_load/static/src/js/pos_picking_load.js

```diff
@@ -0,0 +1,329 @@
+                        'error-traceback', {
+                            message: _t('Unknown Partner'),
+                            comment: _t('Unable to load this picking because the partner' + 
+                                    ' is not known in the Point Of Sale as a customer')
+                        });
+                    picking_selectable = false;
+                }
+
+                if (picking_selectable){
+                    self.$el.find('span.button.validate').show();
+                }
+                else{
+                    self.$el.find('span.button.validate').hide();
+                }
+
+            }).fail(function (error, event){
+                if (parseInt(error.code) === 200) {
+                    // Business Logic Error, not a connection problem
+                    self.pos_widget.screen_selector.show_popup(
+                        'error-traceback', {
+                            message: error.data.message,
+                            comment: error.data.debug
+                        });
+                }
+                else{
+                    self.pos_widget.screen_selector.show_popup('error',{
+                        message: _t('Connection error'),
+                        comment: _t('Can not execute this action because the POS is currently offline'),
+                    });
+                }
+                event.preventDefault();
+            });
+        },
+
+        search_pickings: function(query) {
+            var self = this;
+            var pickingModel = new instance.web.Model(this.model);
+            return pickingModel.call('search_pickings_for_pos', [query || ''])
+            .then(function (result) {
+                self.render_list(result);
+            }).fail(function (error, event){
+                if (parseInt(error.code) === 200) {
+                    // Business Logic Error, not a connection problem
+                    self.pos_widget.screen_selector.show_popup(
+                        'error-traceback', {
+                            message: error.data.message,
+                            comment: error.data.debug
+                        }
+                    );
+                }
+                else{
+                    self.pos_widget.screen_selector.show_popup('error',{
+                        message: _t('Connection error'),
+                        comment: _t('Can not execute this action because the POS is currently offline'),
+                    });
+                }
+                event.preventDefault();
+            });
+        },
+
+        on_click_picking: function(event){
+            this.load_picking(parseInt(event.target.parentNode.dataset.pickingId));
+        },
+
+        render_list: function(pickings){
+            var self = this;
+            var contents = this.$el[0].querySelector('.picking-list-contents');
+            contents.innerHTML = "";
+            for (var i = 0, len = pickings.length; i < len; i++){
```

Comment:
(just for the sytle: you can use `.forEach()`  or `.map()` here instead of for loop.
And in javascript the scope delimited by functions and not {})